### ISSUE TITLE
SR-18-0000156 EN Update the phrase

### ISSUE THREAD

**Issue by RikiSjachrani-COM** [City-of-Madison/Development-Team-2#31]
EN Update the phrase "Street Encroachment permit" to "Encroachment Agreement, Lease, License, or other applicable Agreement must be attached" under the section "ENCROACHMENT INFORMATION" - 

The change was requested from Engineering and we need to apply it to the SSRS Version of the report. 
I'm attaching the Permit from production and will recreate it in Dev for comparison. The production report is currently a Crystal Report, so we are also converting this over. It looks like Chris originally did the conversion, so I'll loop him in on this project

Report Manger name = Excavate in Right of Way Permit
SSRS Report name = /Engineering/PermitToExcavateInPublicROW

[Engineering_PermitToExcavateInPublicROW_20180515_084426.pdf](https://github.com/City-of-Madison/Development-Team-2/files/2010539/Engineering_PermitToExcavateInPublicROW_20180515_084426.pdf)

**Comment by RikiSjachrani-COM** [comment]
Riki 5/16/2018 - Update to scope. They would like the phrase to be updated to "Encroachment Agreement, Lease, License, or other applicable Agreement must be attached"

**Comment by ANavaGomez-COM** [comment]
Working on it.

**Comment by ANavaGomez-COM** [comment]
@RikiSjachrani-COM  - This is update is done.

**Comment by ANavaGomez-COM** [comment]
@RikiSjachrani-COM - I will send a meeting request to test the reports I modified yesterday.
Here are a couple of ALTIds to use for testing:

`--encroachment
ENGROW-2016-00001
--fire order
FIRELV-2014-00183 `

Thanks.

**Comment by ANavaGomez-COM** [comment]
@RikiSjachrani-COM - Riki I've updated the PERMIT TO EXCAVATE IN PUBLIC RIGHT-OF-WAY report to replaced the Crystal version.

**Comment by RikiSjachrani-COM** [comment]
@ANavaGomez-COM Please promote the following SR to test and I’ll notify the customer it's ready for them to UAT
![image](https://user-images.githubusercontent.com/13784495/40319823-1bde9878-5cef-11e8-9ae3-b5b2f3368fb8.png)


Thanks,

Riki

**Comment by ANavaGomez-COM** [comment]
@RikiSjachrani-COM - I cannot find the SR for the work done on the Restaurant Sewer Meter Permit report.

**Comment by RikiSjachrani-COM** [comment]
It's a part of SR-17-0000280 Engineering Restaurant Class Billing

**Comment by ANavaGomez-COM** [comment]
@RikiSjachrani-COM  - Thank you, I found it. 

I have a question, what is the next workflow status, is it "Testing in Progress"?

**Comment by RikiSjachrani-COM** [comment]
I found one other formatting concern. When I selected the option for "Existing" on the connection type, it added another line.
![image](https://user-images.githubusercontent.com/13784495/40320944-06a31944-5cf3-11e8-93ec-3914212c44ca.png)
The testing in progress is usually reserved with larger request. The next step you would enter is "Ready to Test". Once I contact the Customer, I will update it to Notified Customer, and there it will sit for up to a week
